src/Main.java

```diff
@@ -0,0 +1,172 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+//На базе подходящего контейнерного класса разработать класс «Телефонный справочник», который бы содержал
+// произвольное количество абонентов. Для каждого абонента известна фамилия и один номер телефона. Фамилии
+// разных абонентов могут быть одинаковыми, номера телефонов - нет. Справочник должен иметь методы поиска
+// номеров телефонов по фамилии и поиска фамилии по номеру телефона. Базовый контейнерный класс выбрать самостоятельно.
+
+//     1. Добавить в класс «Телефонный справочник» методы сохранения справочника в файле и загрузки справочника из файла.
+//     2. Добавить возможность пополнения справочника новыми абонентами с клавиатуры.
+
+
+//Класс Main реализует «Телефонный справочник» на основе контейнера HashMap<String, String>
+//в качестве ключа используется номер телефона в виде строки, а в качестве значения - фамилия
+//класс включает в себя метод main который запускает программу, а также включены методы:
+//add - добавляет запись по заданным номеру телефона и фамилии
+//del - удаляет запись по номеру телефона
+//save - сохраняет в текстовом файле phone.txt
+//load - загружает из текстового файла phone.txt
+//PrintPhonebook - выводит на екран все записи
+//FindSurname - производит поиск фамилии по номеру телефона
+//FindNumberPhone - производит поиск списка номеров по фамилии
+
+public class Main  {
+    private static Map<String, String> string = new HashMap<String, String>();
+
+    //addPB - добавляет запись по заданным номеру телефона и фамилии
+    private static void add(String phone, String name) {
+        string.put(phone, name);
+    }
+
+    //delPB - удаляет запись по номеру телефона
+    private static void del(String phone) {
+        string.remove(phone);
+    }
+
+    //savePB - сохраняет БД в текстовом файле phone.txt
+    private static void save() throws IOException {
+        BufferedWriter writer = new BufferedWriter(new FileWriter(new File("phone.txt")));
+        for(Map.Entry<String,String> k: string.entrySet()){
+            writer.write(k.getKey() + " " + k.getValue()+System.lineSeparator());
+        }
```

Comment:
Never use one letter variable name (except for i loop):
```suggestion
        for(Map.Entry<String,String> entry: string.entrySet()){
            writer.write(entry.getKey() + " " + entry.getValue() + System.lineSeparator());
        }
```
And don't forget about spaces between symbols.